.github/workflows/container-build.yml

```
@@ -0,0 +1,49 @@
+name: container build
+on:
+  push:
+    tags:
+      - '**'
+    branches:
+      - '**'
+      - '!master'
+  schedule:
+    - cron: '0 0 * * *'
+jobs:
+  container-build:
+    runs-on: ubuntu-latest
+    permissions:
+      packages: write
+    steps:
+      - name: checkout
+        uses: actions/checkout@v3
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+      - name: Log into registry ${{ inputs.registry }}
+        uses: docker/login-action@v2
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
```

I have added the creds for the docker hub into the secrets. @escapedcat I did send you all the details, let me know if you don't have then, and I will send them over again. @kaiehrhardt I am supper sorry about the delay to this.

https://hub.docker.com/r/commitlint/commitlint

```suggestion
        with:
          username: ${{ secrets.DOCKERHUB_USERNAME }}
          password: ${{ secrets.DOCKERHUB_TOKEN }}
```

---

Thanks Ade! 👏

---

Thx + done 👏

---

> @escapedcat I did send you all the details, let me know if you don't have then, and I will send them over again. 

Found them. Sorry. Forgot about these. 🤦